app/controllers/application_controller.rb

```diff
@@ -1,12 +1,27 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
-  helper_method :current_user
+  helper_method :current_user,
+                :one_day_old?,
+                :three_days_old?,
+                :localtime
 
   def current_user
     @current_user ||= User.find(session[:user_id]) if session[:user_id]
   end
 
+  def localtime(meal)
+    meal.created_at.localtime.to_datetime.strftime("%a, %b %d at %l:%M%P")
+  end
+
+  def one_day_old?(meal)
+    (meal.updated_at.to_i - DateTime.now.to_i).abs >= 86400
+  end
+
+  def three_days_old?(meal)
+    (meal.created_at.to_i - DateTime.now.to_i).abs >= 259200
+  end
+
```

Comment:
I like these methods, but I think they belong in the meal model. That way we can also unit test them.

You might also consider renaming the one_day_old? and three_day_old? methods, because by the method names alone I would assume that they evaluate if the meal is exactly one day old, not greater than one day old.

Reply:
I agree with your thought here. Moving these to the meal model makes sense and being able to unit test them is important. I also second the idea of renaming them. The current names versus what they actually do is a little confusing and adding a greater_than in front of the names would help.